### PR TITLE
Fix logging_enabled param initial loading on Linux

### LINKED ISSUE #7167: Toggle which enable logging works from second attempt
### Description

*Type*: Bug
*Summary*: when you enable logging, it is not enabled from the first time - so you should log in again and enable it second time. Reproducible on Linux and Windows. Couldn't reproduce on Mac OSx.

#### Expected behavior
can enable logging from the first attempt

#### Actual behavior
logging is enabled only from second attepmt

### Reproduction

- Open Status
- Log in
- Enable logging
- Log in and check logging state

### Additional Information
[comment]: # (Please do your best to fill this out.)
* Status version: [release #9](https://ci.status.im/job/status-react/job/release/job/release%252Fdesktop%252F0.9.0/9/)
* Operating System: Windows, Linux
* Video: http://take.ms/JqcQE

### PR DESCRIPTION
**Pull request by vitvly**
Fixes #7167

### CODE REVIEW

File: src/status_im/init/core.cljs [hunk 1]
```diff
@@ -46,7 +46,9 @@
 (defn restore-native-settings! []
   (when platform/desktop?
     (.getValue rn-dependencies/desktop-config "logging_enabled"
-               #(re-frame/dispatch [:set-in [:desktop/desktop :logging-enabled] %1]))))
+               #(re-frame/dispatch [:set-in [:desktop/desktop :logging-enabled]
+                                    (if (boolean? %1)
```
Comment (rasom) [inline src/status_im/init/core.cljs:50]: can it be just `%`?

**Review by rasom — APPROVED** [review]

**Review by flexsurfer — APPROVED** [review]

**Review by vkjr — APPROVED** [review]

**Comment by churik** [comment]
Tested on Mac OS, Linux and Windows. Also checked Notifications toggle - it doesn't affect to Logging toggle.

**Comment by antdanchenko** [comment]
@churik I am going to re-run `test_messaging_in_different_networks` in order to unblock the PR

**Comment by churik** [comment]
@siphiuel  please, merge it also in `release/desktop/0.9.0`

**Comment by vitvly** [comment]
Done.

**Pull request merged**